models/stg_quickbooks__credit_card_payment_txn.sql

```diff
@@ -0,0 +1,40 @@
+--To enable this model, set the using_invoice_bundle variable within your dbt_project.yml file to True.
+{{ config(enabled=var('using_credit_card_payment_txn', False)) }}
+
+with base as (
+
+    select * 
+    from {{ ref('stg_quickbooks__credit_card_payment_txn_tmp') }}
+),
+
+fields as (
+
+    select
+        {{
```

Comment:
I know we've included this text before, do we need it here? 

> 
>         /*
>         The below macro is used to generate the correct SQL for package staging models. It takes a list of columns 
>         that are expected/needed (staging_columns from dbt_quickbooks_source/models/tmp/) and compares it with columns 
>         in the source (source_columns from dbt_quickbooks_source/macros/).
>         For more information refer to our dbt_fivetran_utils documentation (https://github.com/fivetran/dbt_fivetran_utils.git).
>         */

Reply:
No need to include that. We are actually considering removing it across all our packages.